assets/js/blocks/filter-wrapper/edit.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps, InnerBlocks } from '@wordpress/block-editor';
+import type { BlockEditProps } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { Attributes } from './types';
+
+const Edit = ( { attributes }: BlockEditProps< Attributes > ) => {
+	const blockProps = useBlockProps();
+
+	return (
+		<div { ...blockProps }>
+			<InnerBlocks
+				allowedBlocks={ [ 'core/heading' ] }
+				template={ [
+					[
+						'core/heading',
+						{ level: 3, content: attributes.heading || '' },
```

Comment:
`level: 3` does this mean that even if the user has the title level heading set to Level 2 _before_ upgrading that after the upgrade this will be changed to 3? If so, is there any reason we settled on level 3 heading?

Reply:
Right, we should use the current heading level if available.